integrations/slack.mdx

```diff
@@ -1,28 +1,108 @@
 ---
 title: "Slack"
-description: "Create and receive feedback directly in Slack. Share weekly summary reports with the team."
+description: "Integrate Monterey.ai with your Slack workspace"
 ---
 
-![slack](https://monterey-documentation-gifs.s3.us-east-2.amazonaws.com/slack.gif)
+![slack integration cover](/images/slack-integration-cover.png)
+
+## Overview
+
+The Slack integration makes it effortless to stay on top of customer feedback and insights directly within your team's workspace. With the integration, your team can:
+
+- Uncover insights from channels shared with customers
+- Interact with [Ask Monty](/ask-monty) via chat conversations
+- Get alerts in chosen Slack channels for new feedback, theme reports, and weekly digests
+- Create feedback directly from Slack messages
+
+## Setup
+
+To get started, navigate to the **[integration setting page](https://app.monterey.ai/w/default/settings/integrations/)** and select **Slack**.
+
+### Connect Workspace
+
+<Steps>
+  <Step title='Click "Connect Monterey.ai to Slack"'></Step>
+  <Step title="Choose Features">
+    We only ask for the permissions that are necessary for the features you plan
+    to use.
+  </Step>
+  <Step title='Click "Connect"'>
+    This will take you to Slack to review and approve the permissions we
+    request.
+  </Step>
+</Steps>
```

Comment:
Oh sweet, I'm going to go back to the PRs I have open and use these components. When I'm in these markdown files I just don't even think of the fact that it's mdx/has some components available.